userland/libtock/rng.h

```diff
@@ -0,0 +1,50 @@
+#pragma once
```

Comment:
What do you mean, `#pragma once` is well known? It's the replacement for include guards in modern C[++] ?